modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/AccountResourceTest.java

```diff
@@ -869,6 +892,84 @@ private void _testPatchAccountWithContactInformation() throws Exception {
 			patchAccount.getAccountContactInformation());
 	}
 
+	private void _testPatchAccountWithMoreExternalReferenceCodes()
+		throws Exception {
+
+		Account postAccount = testPatchAccount_addAccount();
+
+		Account parentAccount = randomAccount();
+		Account randomPatchAccount = randomPatchAccount();
+
+		Account postParentAccount = testPostAccount_addAccount(parentAccount);
+
+		Organization organization1 = OrganizationTestUtil.addOrganization();
+		Organization organization2 = OrganizationTestUtil.addOrganization();
+		Address billingAddress = OrganizationTestUtil.addAddress(organization1);
+		Address shippingAddress = OrganizationTestUtil.addAddress(
+			organization2);
+
+		FileEntry fileEntry = _addFileEntry();
+
+		randomPatchAccount.setDefaultBillingAddressExternalReferenceCode(
+			billingAddress.getExternalReferenceCode());
+		randomPatchAccount.setDefaultBillingAddressId(0L);
+
```

Comment:
check if the empty lines are needed otherwise remove them (like you did in the post test)